tsconfig.json

```diff
@@ -22,21 +16,14 @@
     "rootDir": "src",
     "target": "es2015",
     "paths": {
-      "Assets/*": [
-        "./Assets/*"
-      ]
+      "Assets/*": ["./Assets/*"]
     }
   },
   "include": [
+    "node_modules/styled-components/typings/styled-components.d.ts",
```

Comment:
oh cool, i remember that for a while they didn't have a ts definition

Reply:
Would it be possible to use this property rather than include? https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types